packages/frontend/src/config/envParsers.js

```diff
@@ -12,7 +12,7 @@ const parseObjectFromShell = (envVal) => {
 };
 
 const parseCommaSeperatedStringAsArrayFromShell = (envVal) =>
-    envValIsSet(envVal) ? envVal.split(",") : undefined;
+    envValIsSet(envVal) ? envVal.replace(/\s/g, "").split(",") : undefined;
```

Comment:
We don't want this to always strip all whitespace from any env vars we're using `parseCommaSeperatedStringAsArrayFromShell` with -- it's just _coincidental_ that none of our current comma-separated values don't include spaces.  Rather than put this here, please run the .replace() in-line just for the `SMS_BLACKLIST` env var in `configFromEnvironment`, then pass the resulting 'cleaned' string into this method (which should only split on comma with the `envValIsSet` guard)

Reply:
Done! 😄